src/functions/types.ts

```diff
@@ -396,3 +397,32 @@ export type RuntimeUnhandledEventContext<
     // deno-lint-ignore no-explicit-any
     body: any;
   };
+
+type DefineFunctionReturnType = ReturnType<typeof DefineFunction>;
+
+type FunctionRuntimeReturnArgs<
```

Comment:
Can we add a comment above here to summarize what this type is for? If I understand correctly, it simply it just unwraps the return from a function in case it is wrapped in a promise.

Reply:
Yes exactly this simply unwraps the promise, added a comment 💯